vm/src/builtins/int.rs

```diff
@@ -570,11 +570,11 @@ impl PyInt {
     }
 
     #[pymethod(name = "as_integer_ratio")]
-    fn as_integer_ratio(&self, vm: &VirtualMachine) -> PyResult {
-        Ok(vm.ctx.new_tuple(vec![
+    fn as_integer_ratio(&self, vm: &VirtualMachine) -> PyObjectRef {
+        vm.ctx.new_tuple(vec![
             vm.ctx.new_bigint(&self.value),
```

Comment:
This could maybe return `(PyIntRef, u8)`, just `(zelf, 1)`

Reply:
it causes test_long error for bool type